### PR TITLE
debian: silence couple warnings

### PR DESCRIPTION
**Pull request by tchaikov**

### CODE REVIEW

**Comment by tchaikov** [comment]
@dmick here are more `debian/rules` cleanups.

**Comment by tchaikov** [comment]
rebased against master.

**Comment by tchaikov** [comment]
@dmick ping?

File: debian/control [hunk 1]
```diff
@@ -649,7 +648,6 @@ Section: python
 Depends: python-rados (= ${binary:Version}),
          python-rbd (= ${binary:Version}),
          python-cephfs (= ${binary:Version})
-X-Python-Version: >= 2.6
```
Comment (dmick) [inline debian/control]: Is this true for versions as old as Ubuntu Trusty?

Comment (tchaikov) [inline debian/control]: see http://packages.ubuntu.com/trusty/python

Comment (dmick) [inline debian/control]: kefu answers in PM that yes it si

**Pull request merged**